.learn/resets/04.1-Combined-Rules/styles.css

```diff
@@ -0,0 +1,14 @@
+.what {
+	width: 50px;
+	height: 50px;
+	padding-top: 10px;
```

Comment:
Can you please remove the `.learn/resets` folder? Before you commit on the exercises make sure the resets folders is deleted.

Reply:
done